apps/examples/wifi_manager/wm_test/wm_test.c

```diff
@@ -888,9 +972,9 @@ int main(int argc, FAR char *argv[])
 #else
 int wm_test_main(int argc, char *argv[])
 #endif
-{
-	WT_LOG(TAG, "wifi manager test!!");
-	int res = _wt_signal_init();
+{	
```

Comment:
Extra tab spaces at the end of the text in a particular line need to be checked and removed please

Reply:
Done